django/core/checks/compatibility/django_1_10.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import unicode_literals
+
+from django.conf import global_settings, settings
+
+from .. import Tags, Warning, register
+
+
+@register(Tags.compatibility)
+def check_duplicate_middleware_settings(app_configs, **kwargs):
+    if settings.MIDDLEWARE is not None and settings.MIDDLEWARE_CLASSES != global_settings.MIDDLEWARE_CLASSES:
+        return [Warning(
+            "The MIDDLEWARE_CLASSES setting is deprecated in Django 1.10 "
+            "and the MIDDLEWARE setting takes precedence. The value of "
```

Comment:
Maybe prepend final sentence with "Since you have set MIDDLEWARE, " -- otherwise it kind of reads as if MIDDLEWARE_CLASSES is _always_ ignored in 1.10.